packages/graphql-transformers/src/openCrudWhereTransform/manyRelationConditionSubquery.ts

```diff
@@ -42,6 +42,6 @@ function removeIncludesAttributes(includes) {
   }
 }
 
-module.exports = {
+export default {
   buildConditionSubquery
```

Comment:
it is not the same as export const buildConditionSubquery = ()=> ... ?

Reply:
Kindof, but since it was until now with module.exports, we got the two flavours of this one happening.
const { buildConditionSubquery } = require('something') and const something = require('something')
Both will work with module.exports as the destruct will come in our favour, however, with import export it doesn't happen as we got export default & named export